app/controllers/api/v1/rents_controller.rb

```diff
@@ -20,6 +21,10 @@ def create
         end
       end
 
+      def set_locale
+        I18n.locale = current_user.locale
```

Comment:
```suggestion
        I18n.locale = current_user&.locale || I18n.default_locale
```
Look this!! http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/